content/en/docs/protocol/aqua-identity-protocol.md

```diff
@@ -357,13 +357,12 @@ Registration Event: 1
 
 -   Locally completed (within the Self-Issued Identity Claim) by
     changing the status to 'revoked' and signing by the issuer. This
-    adds a receipt to the identity claim which is displayed when
+    adds a receipt to the identity claim, which is displayed when
     verifying the claim. This includes the revocation transaction for
     the revocation on the
     [Identity-Registry](https://pkc.inblock.io/index.php/Identity-Registry)
 -   If there is a new claim which is succeeding the previous identity
-    claim[Claim
-    Registry](https://pkc.inblock.io/index.php/Claim_Registry), then
+    claim[Claim Registry](https://pkc.inblock.io/index.php/Claim_Registry), then
```

Comment:
The older version is fine. The new line is caused by the wrapping to fit a 100-char width.